LinoBarreca/Marlin
# Multiple SPI and Smart SD handling - test

We will use this place to have a feedback from testers.
**_SKR-PRO ONLY_**
(other boards are broken at the moment, don't waste your time trying to build it)

We need to test the following situations:
Printer without any sd card inserted.
Screen with sd-slot and card **in screen** sd-slot.
Screen with sd-slot and card on board.
Screen without sd-slot and card on board.

at the moment `LULZBOT_TOUCH_UI` and `MAX6675_SEPARATE_SPI` are broken.
if you have that DON'T use this version.

How to test:
Pull the sources.
Add your config files.
Compile.
Put the firmware in the onboard sd.
Put a .gcode in the onboard sd.
Turn on the printer.
Check that you can see the gcode.
